apps/interactive-calibration/main.cpp

```diff
@@ -180,7 +180,7 @@ int main(int argc, char** argv)
                 }
                 else {
 #ifdef HAVE_OPENCV_ARUCO
```

Comment:
We always have Aruco since now.

Reply:
There is `interpolateCornersCharuco()` call in frameProcessor.cpp:

```
cv::aruco::detectMarkers(frame, cv::makePtr<cv::aruco::Dictionary>(mArucoDictionary), corners, ids, cv::makePtr<cv::aruco::DetectorParameters>(), rejected);
    cv::aruco::refineDetectedMarkers(frame, board, corners, ids, rejected);
    cv::Mat currentCharucoCorners, currentCharucoIds;
    if(ids.size() > 0)
        cv::aruco::interpolateCornersCharuco(corners, ids, frame, mCharucoBoard, currentCharucoCorners,
                                         currentCharucoIds);
```

new API cannot resolve old API dependencies